src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        List<Ticket> tickets = shoppingCart.getTickets();
+        User user = shoppingCart.getUser();
+        LocalDateTime localDateTime = LocalDateTime.now();
+        Order newOrder = new Order(tickets, localDateTime, user);
```

Comment:
pass a copy of tickets
![image](https://github.com/mate-academy/hibernate-order-hw/assets/49767049/a0909541-45a4-4b37-acf5-d8cb42986125)